apps/nextjs-website/react-components/components/Header/Header.tsx

```diff
@@ -0,0 +1,124 @@
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { useEffect, useState } from 'react';
+import { HamburgerMenu } from './components/HamburgerMenu';
+import { Navigation } from './components/Navigation';
+import { Content } from './components/Title';
+import { HeaderProps } from '@react-components/types/Header/Header.types';
+import { CtaButtons } from '../common/Common';
+import { BackgroundColor } from '../common/Common.helpers';
+
+const Header = ({
+  product,
+  theme,
+  menu,
+  ctaButtons,
+  avatar,
+  beta,
+  logo,
+}: HeaderProps) => {
+  const [headerOpen, setHeaderOpen] = useState(false);
+
+  const openHeader = () => {
+    setHeaderOpen(true);
+  };
+
+  const closeHeader = () => {
+    setHeaderOpen(false);
+  };
+
+  const onResize = () => {
+    closeHeader();
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', onResize);
+
+    return () => {
+      window.removeEventListener('resize', onResize);
+    };
+  }, []);
+
+  const backgroundColor = BackgroundColor(theme);
+
+  const HeaderCtas = () => {
+    return ctaButtons && ctaButtons.length > 0 ? (
+      <Stack direction='row'>
+        {CtaButtons({
+          ctaButtons: ctaButtons.map((CtaButton) => ({
+            ...CtaButton,
+            sx: { width: { md: 'auto', xs: '100%' } },
+          })),
+          theme,
+        })}
+      </Stack>
+    ) : null;
+  };
+
+  return (
+    <Box
+      bgcolor={backgroundColor}
+      paddingX={{ xs: 1, sm: 3 }}
+      component='header'
+      role='banner'
+    >
+      <Stack
+        direction={{ md: 'row' }}
+        paddingY={{ xs: 2, sm: 3, md: 0 }}
+        gap={4}
+      >
+        <Stack
+          sx={{
+            flexDirection: 'row',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+          }}
+        >
+          <Content
+            theme={theme}
+            product={product}
+            {...(beta ? { beta } : {})}
+            {...(avatar ? { avatar } : {})}
+            {...(logo ? { logo } : {})}
+          />
+
+          <Stack
+            sx={{ display: { md: 'none' } }}
+            direction='row'
+            alignItems='center'
+            gap={4}
+          >
+            <Box sx={{ display: { xs: 'none', sm: 'block', md: 'none' } }}>
+              <HeaderCtas />
+            </Box>
```

Comment:
```suggestion
            <HeaderCtas  sx={{ display: { xs: 'none', sm: 'block', md: 'none' } }}/>
```
Get rid of Box and make `sx` a prop for HeaderCtas

Reply:
Will be resolved with task [B2BP-580](https://pagopa.atlassian.net/browse/B2BP-580?atlOrigin=eyJpIjoiZTlhODA0Y2RmY2JiNGZlZWJkMmUyNDQwYjljNmQzZjIiLCJwIjoiaiJ9)